css/bootstrap.css

```diff
@@ -6758,3 +6771,43 @@ button.close {
   }
 }
 /*# sourceMappingURL=bootstrap.css.map */
+
+
+footer
+{
+    background-color:#4A148C;
+    color: #9649CB;
+    clear:both;
+    text-align:left;
+    padding:2%;
+    border-top: 10px;
+    margin-right: 0;
+    margin-left: 0;
+    margin-bottom: 0;
+    padding-left: 25%;
+    padding-right: 25%;
+    padding-bottom: 8%;
+    height: 50px;
+    bottom: 0;
+    display: block;
+    z-index: -1;
+    height: 100px;
+}
+
+/*html,
+body {
+  margin:0;
+  padding:0;
+  height:100%;
+}
+#wrapper {
+  min-height:100%;
+  position:relative;
+}
+#header {
+  background:#ededed;
+  padding:10px;
+}
+#content {
+  padding-bottom:100px; /* Height of the footer element */
+  
```

Comment:
Please do not leave comments or code that is not needed. We want to keep the repo clean as possible!